website/gatherDocs.js

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+const fs = require("fs-extra");
+const git = require("nodegit");
+const execSync = require("child_process").execSync;
+const { major, lt } = require("semver");
+const REPO_URL = "https://github.com/wix/detox.git";
+
+async function getVersions() {
+  const tmp = fs.mkdtempSync("detox-versions");
+  const repo = await git.Clone(REPO_URL, tmp);
+  const tags = await git.Tag.list(repo);
+
+  const semverTags = tags
+    .filter(tag => !tag.includes("@"))
+    .filter(tag => tag.split(".").length === 3 && major(tag) >= 7)
+    .sort()
+    .reverse();
+  await fs.remove(tmp);
+  return semverTags;
+}
+
+const sidebars = [];
+
+async function cleanupExistingVersions() {
+  console.log("Cleanup versioned docs");
+  await fs.remove("./versions.json");
+  await fs.emptyDir("./versioned_docs");
+  await fs.emptyDir("./versioned_sidebars");
+}
+
+// https://stackoverflow.com/questions/29569913/switch-branch-tag-with-nodegit?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa
+function checkOutTag(repo, tag) {
```

Comment:
LOL, utm_source will provide false data for trackers next time someone opens this link ;p
JK, but the query string can be removed

Reply:
Right, I'll get rid of that and see if I can get a better permalink